esp/logging/logginglib/datafieldmap.cpp

```diff
@@ -33,7 +33,7 @@ void readLogGroupCfg(IPropertyTree* cfg, StringAttr& defaultLogGroup, MapStringT
     Owned<IPropertyTreeIterator> iter = cfg->getElements("LogGroup");
     ForEach(*iter)
     {
-        ensureInputString(iter->query().queryProp("@name"), true, groupName, -1, "LogGroup @name required");
+        ensureInputString(iter->query().queryProp("@Name"), true, groupName, -1, "LogGroup @Name required");
```

Comment:
Rodrigo suggested camel casing for consistency, but NOT leading capitals... please revert this